CHANGELOG.md

```diff
@@ -1,5 +1,24 @@
 # Azure Kubernetes Service Changelog
 
+## Release 2021-09-30
+
+This release is rolling out to all regions - estimated time for completed roll out is 2021-10-07 for public cloud and 2021-10-14 for sovereign clouds.
+
+### Announcement
+
+* In order to preserve any deallocated VMs, you must to set Scale-down Mode to `Deallocate`. That includes VMs that have been deallocated using IaaS APIs (Virtual Machine Scale Set APIs). Setting Scale-down Mode to `Delete` will remove any deallocated VMs.
```

Comment:
Do we still need this announcement? it has been in maybe the last 3 releases?
The AZ AKS addon cli commands have finally been merged, might be good to have that in the announcements instead.
Just a thought.